src/onegov/org/templates/layout.pt

```diff
@@ -84,13 +84,16 @@
       <main>
 
         <header tal:define="header_options request.app.org.header_options or {}" >
-          <div tal:condition="header_options.get('left_header_announcement', '')"
-               id="announcement"
-               style="color: ${header_options.get('left_header_announcement_font_color', '#000000')};
-                      background-color: ${header_options.get('left_header_announcement_bg_color', '#FBBC05')};">
-              <div class="grid-container">
-                ${header_options['left_header_announcement']}
-              </div>
+          <div tal:condition="header_options.get('announcement', '') and (request.is_manager or not header_options.get('announcement_is_private'))"
```

Comment:
Description in the settings is "not logged in" but here it's is_manager?

Reply:
But the description is "Only show Announcement for logged-in users"?